src/drivers/drv_hrt.h

```diff
@@ -235,6 +235,12 @@ __EXPORT extern void	hrt_call_delay(struct hrt_call *entry, hrt_abstime delay);
  */
 __EXPORT extern void	hrt_init(void);
 
+#if defined(CONFIG_BUILD_FLAT)
+#  define hrt_ioctl_init()
```

Comment:
This define is probably not needed? The call is already flagged out in px4_init

Reply:
Sure, either can be removed (the flagging on the caller side or here). Which is btw the preferred way in this project ?

Reply:
Seems like this file does not have any macro definitions for unused functions so kept the flagging in px4_init